.circleci/scripts/install-linux-csharp.sh

```diff
@@ -10,5 +8,4 @@ sudo apt-get update; \
   sudo apt-get update && \
   sudo apt-get install -y dotnet-sdk-3.1
 export PATH=$PATH:~/.dotnet
-echo "done installing .Net SDK"
```

Comment:
Probably OK, but it looks like this is missing a new line on the end, which annoys UNIX.